### DISCUSSION TITLE
Access restricted

### DISCUSSION THREAD

**Discussion by Nilam2903** [community/community#128141] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

hii,i am not able to load information for github.con for my project.i am getting the error stating that access to this site has been restricted.why i am getting this error?

**Comment by dev-ugurkontel** [comment]
I list a few possible steps to solve the problem;

- Restart your computer.

- Change the DNS addresses you use.

- Reset your network settings.

- Reset your modem to factory settings.

- Ask for technical support from your service provider.
- 
![051321_1940_Windows10Ne1](https://github.com/community/community/assets/53796844/9a78db6d-c2f0-4176-b513-d7f8faa32942)
![051321_1940_Windows10Ne2](https://github.com/community/community/assets/53796844/dd1a45f5-b467-4bd9-b434-d8114aa20d45)
![051321_1940_Windows10Ne3](https://github.com/community/community/assets/53796844/68746638-b61f-490d-a075-9d675c3ce45e)
![051321_1940_Windows10Ne4](https://github.com/community/community/assets/53796844/cf688058-f213-48d0-9975-d224a9710bd9)
![051321_1940_Windows10Ne5](https://github.com/community/community/assets/53796844/81c6617e-3f26-41d5-b020-890ad7973f91)

**Reply by ebndev** [reply]
Hi @dev-ugurkontel, thanks for taking the time to answer!

@Nilam2903 if the response above helped, please feel encouraged to mark it as the answer. If not, let us know if you have further questions or post the solution if you've resolved it on your own. It would be helpful for other users who might experience this in the future 😄
